### ISSUE TITLE
Doesn't seem to resolve 26.0.1 version on some windows systems

### ISSUE THREAD

**Issue by voxelbusters** [googlesamples/unity-jar-resolver#82]
On some windows system it doesn't seem to resolve 26.0.1 but only 26.0.0-alpha1 support libraries only.

```
No compatible versions of com.android.support:support-v4:26.0.1+ found given the set of required dependencies.

com.android.support:support-v4:26.0.1+ was referenced by:
   at System.Environment.get_StackTrace() in /Users/builduser/buildslave/mono/build/mcs/class/corlib/System/Environment.cs:line 227
   at Google.JarResolver.Dependency..ctor(System.String group, System.String artifact, System.String version, System.String[] packageIds, System.String[] repositories, System.String createdBy)
   at Google.JarResolver.PlayServicesSupport.AddCommonPackageIds(Google.JarResolver.Dependency dep)
   at Google.JarResolver.PlayServicesSupport.DependOn(System.String group, System.String artifact, System.String version, System.String[] packageIds, System.String[] repositories, System.String createdBy)
   at System.Reflection.MonoMethod.InternalInvoke(System.Object , System.Object[] , System.Exception ByRef )
   at System.Reflection.MonoMethod.Invoke(System.Object obj, BindingFlags invokeAttr, System.Reflection.Binder binder, System.Object[] parameters, System.Globalization.CultureInfo culture) in /Users/builduser/buildslave/mono/build/mcs/class/corlib/System.Reflection/MonoMethod.cs:line 222
   at System.Reflection.MethodBase.Invoke(System.Object obj, System.Object[] parameters) in /Users/builduser/buildslave/mono/build/mcs/class/corlib/System.Reflection/MethodBase.cs:line 115
   at Google.VersionHandler.InvokeMethod(System.Type type, System.Object objectInstance, System.String methodName, System.Object[] args, System.Collections.Generic.Dictionary`2 namedArgs)
   at Google.VersionHandler.InvokeInstanceMethod(System.Object objectInstance, System.String methodName, System.Object[] args, System.Collections.Generic.Dictionary`2 namedArgs)
   at VoxelBusters.NativePlugins.NPAndroidLibraryDependencies.CreateDependencies() in D:\Unity Personal Projects\TiltanFinalProject\Assets\VoxelBusters\NativePlugins\Editor\NPAndroidLibraryDependencies.cs:line 110
   at VoxelBusters.NativePlugins.NPAndroidLibraryDependencies.<NPAndroidLibraryDependencies>m__0() in D:\Unity Personal Projects\TiltanFinalProject\Assets\VoxelBusters\NativePlugins\Editor\NPAndroidLibraryDependencies.cs:line 42
   at VoxelBusters.Utility.EditorUtils.ManageActiveInvokes() in D:\Unity Personal Projects\TiltanFinalProject\Assets\Standard Assets\VoxelBusters\Common\Utility\UnityEditor\Scripts\Invoke\EditorUtils.cs:line 101
   at VoxelBusters.Utility.EditorUtils.EditorUpdate() in D:\Unity Personal Projects\TiltanFinalProject\Assets\Standard Assets\VoxelBusters\Common\Utility\UnityEditor\Scripts\Invoke\EditorUtils.cs:line 83
   at UnityEditor.EditorApplication.Internal_CallUpdateFunctions() in C:\buildslave\unity\build\Editor\Mono\EditorApplication.cs:line 109


UnityEngine.Debug:LogError(Object)
GooglePlayServices.PlayServicesResolver:LogDelegate(String, LogLevel)
Google.JarResolver.PlayServicesSupport:Log(String, LogLevel, Boolean)
Google.JarResolver.PlayServicesSupport:GetDependencies(Dependency, List`1, Boolean)
Google.JarResolver.PlayServicesSupport:GetTransitiveDependencies(Dictionary`2, List`1, Boolean)
GooglePlayServices.<DoResolutionUnsafe>c__AnonStorey18:<>m__28(List`1)
GooglePlayServices.ResolverVer1_1:DoResolutionUnsafe(PlayServicesSupport, String, OverwriteConfirmation, Action)
GooglePlayServices.<DoResolution>c__AnonStorey17:<>m__25()
GooglePlayServices.ResolverVer1_1:UpdateTryResolution()
UnityEditor.EditorApplication:Internal_CallUpdateFunctions()

```

**Comment by stewartmiles** [comment]
This is a warning AFAIK.  Are you sure Gradle resolution is enabled on these systems?  If it's using the legacy resolution method (i.e just reading the maven repo in the Android SDK on the machine) this is going to fail.

It doesn't look like the download_artifacts.gradle script was executed by the resolution.

**Comment by voxelbusters** [comment]
@stewartmiles Do you mean Setting Gradle(new) setting in Unity?

**Comment by stewartmiles** [comment]
Yes you need to enable Gradle resolution.  The Android SDK (thing you download to your machine) doesn't have anything other than support library 26.0.0-alpha .   The Android support libraries are now only published to maven.google.com which is only supported by the Gradle resolver.

**Comment by voxelbusters** [comment]
> Yes you need to enable Gradle resolution.

By this do you mean setting export of Unity from Internal(default) to Gradle(new)? Please correct me if I'm wrong.

**Comment by stewartmiles** [comment]
In the Android Resolver settings, under "Assets > Play Services Resolver > Android Resolver > Settings" there is an option to switch between using Gradle vs. legacy resolution.  This is different to Unity's project export settings.  Essentially we have a load of legacy logic that parses the local maven repos in the Android SDK which is now deprecated since most Google packages are moving into Google Maven (maven.google.com).  The legacy logic cannot fetch packages from Google Maven or any other non-local repo.  So I added support for resolution using Gradle which is now the default resolution strategy which supports online maven repos (e.g maven.google.com) as well as offline repos (like the Android SDK).  Is it possible some of your machines have legacy resolution enabled?

**Comment by stewartmiles** [comment]
@voxelbusters any news on whether you're using the legacy resolution step or the gradle build?

**Comment by johnb003** [comment]
In the resolver released with Firebase 4.2.1 we've removed support for the old legacy resolver. Please see if this fixes your issue.

**Comment by stewartmiles** [comment]
We'll assume this has been resolved with the Gradle resolution step.  @voxelbusters please reopen if this isn't the case.
